### PR TITLE
work on build helpers for controllers

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: pkg/build/apis/build/validation/validation.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/openshift/openshift-apiserver/pkg/apiserver/labelselector"
 	buildapi "github.com/openshift/origin/pkg/build/apis/build"
 	buildinternalhelpers "github.com/openshift/origin/pkg/build/apis/build/internal_helpers"
-	buildutil "github.com/openshift/origin/pkg/build/util"
+	buildutil "github.com/openshift/origin/pkg/build/buildutil"
```
Comment (mfojtik) [inline pkg/build/apis/build/validation/validation.go]: @deads2k drop the alias?

File: pkg/build/metrics/prometheus/metrics.go [hunk 2]
```diff
@@ -137,3 +136,17 @@ func (bc *buildCollector) collectBuild(ch chan<- prometheus.Metric, b *buildv1.B
 	}
 	return key
 }
+
+func strategyType(strategy buildv1.BuildStrategy) string {
```
Comment (mfojtik) [inline pkg/build/metrics/prometheus/metrics.go]: @deads2k i was thinking of moving this into library-go

File: pkg/build/webhook/generic/generic.go [hunk 3]
```diff
@@ -7,6 +7,8 @@ import (
 	"mime"
 	"net/http"
 
+	"github.com/openshift/origin/pkg/build/buildutil"
```
Comment (mfojtik) [inline pkg/build/webhook/generic/generic.go]: nit: wrong order

File: pkg/unidling/OWNERS [hunk 4]
```diff
@@ -1,5 +0,0 @@
-reviewers:
```
Comment (mfojtik) [inline pkg/unidling/OWNERS]: @deads2k who owns unidling these days?

**Comment by mfojtik** [comment]
@deads2k if this fail the CI, can you strip the import alias? otherwise we can do this in follow up.

**Comment by soltysh** [comment]
Picked and fixed the rebase problem in https://github.com/openshift/origin/pull/23077. 
/close

**Pull request closed without merging**